sanic/app.py

```diff
@@ -1453,7 +1453,12 @@ def get_app(
                 return cls.get_app("__mp_main__", force_create=force_create)
             if force_create:
                 return cls(name)
-            raise SanicException(f'Sanic app name "{name}" not found.')
+            raise SanicException(
+                f"Sanic app name '{name}' not found.\n"
+                "App instantiation must occur outside "
+                "if __name__ == '__main__' "
+                "block or by using an AppLoader."
```

Comment:
We should add a message to prompt the user to go here: https://sanic.dev/en/guide/deployment/app-loader.html

Reply:
Cool! I'll edit this and update the test tomorrow :)